.github/workflows/publish-demo-app.yml

```diff
@@ -54,12 +54,30 @@ jobs:
       env:
         CLIENT_KEY: ${{ secrets.DEMO_APP_TEST_ENV_CLIENT_KEY }}
         DEMO_SERVER_API_KEY: ${{ secrets.DEMO_SERVER_TEST_ENV_API_KEY }}
+        MERCHANT_ACCOUNT: ${{ secrets.MERCHANT_ACCOUNT }}
+        APPLE_DEVELOPMENT_TEAM_ID: ${{ secrets.APPLE_DEVELOPMENT_TEAM_ID }}
+        ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
```

Comment:
```suggestion
        ENVIRONMENT: 'test'
```

Reply:
its not always test, it can be live also, if we want, right?